src/lib/auth.ts

```diff
@@ -101,6 +104,7 @@ export const ROLES: {
 		"resin.service_instance.get?service_type eq 'vpn'",
 		'resin.service_label.all',
 		'resin.user.read',
+		`resin.user__has__public_key.all?${matchesActor}`,
```

Comment:
`matchesActor` won't work here, as it only has a relationship to the user. It needs to look at the `user` then link to the `actor`